jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -261,7 +302,12 @@ node {
                 when(isGARelease) {
                     gitCommitWhenChanges("Updated ${productName} product version to ${latestRelease}") { msgs ->
                         manifestFileTxt = readFile(installationManifestFile)
-                        manifestFileTxt = manifestFileTxt.replaceFirst(/${manifestVar}: '.*'/, "${manifestVar}: '${latestRelease}'")
+                        if (releaseTagVar != "") {
+                          // Updates the release tag version
+                          manifestFileTxt = manifestFileTxt.replaceFirst(/${releaseTagVar}: '.*'/, "${releaseTagVar}: '${latestRelease}'")
+                        }
+                        // Updates the product version
+                        manifestFileTxt = manifestFileTxt.replaceFirst(/${productVersionVar}: '.*'/, "${productVersionVar}: '${productVersion}'")
```

Comment:
Since we are guarding against releaseTagVar being empty, although i'm not sure why that would ever be the case, should we add the same check around this update for productVersionVar?

Reply:
You seemed to have removed the check entirely? Is this correct? Surely this will fail if productVersionVar is blank which it will be for everything other than webapp?